src/helpers/getAdvices.ts

```diff
@@ -1,11 +1,16 @@
-export const getAdvices = async () => {
+import { Advice, AdviceData } from '../types/index.ts';
+
+export const getAdvices = async (): Promise<Advice> => {
   const url = 'https://api.adviceslip.com/advice';
 
   const response = await fetch(url);
-  const data = await response.json();
+  const data:AdviceData = await response.json();
 
   const {id, advice} = data.slip;
 
-  console.log(`id: ${id}, advice: ${advice}`)
+  return{
+    id,
+    advice
+  }
```

Comment:
Podrías resumir el return con lo siguiente:

```suggestion
  return { ...data.slip }
```